src/autoUpdate.civet

```diff
@@ -1,32 +0,0 @@
-/** Script to automatically upgrade bot
```

Comment:
Also please don't delete this file. Unless you've built in *automatic* updating somewhere that I missed?